pages/en/zkapps/advanced-concepts.mdx

```diff
@@ -0,0 +1,64 @@
+import Page from "@reason/pages/Docs";
+export default Page({
+  title: "Advanced Concepts | Mina Protocol",
+  description: "Check out these advanced concepts regarding zkApps.",
+});
+
+# Advanced Concepts
+
+Build onto what you already learned regarding zkApps and SnarkyJS
+
+This page explores additional topics that requires some familiarity with building zkApps, SnarkyJS, and crytography.
+
+## Events
+
+Events represent arbitrary information that can be passed along with a transaction.
+
```

Comment:
@mitschabaude  Could we add a 2nd sentence or paragraph (no code) that verbally mentions an even easier to grasp use case of events? So that they can get the concept of events and then can transition into your current example: "Let's look at another very common use case...".

Basically, the goal is to make events instantly easy to understand why they would be useful without immediately presenting them with the complexity in the current second paragraph around Merkle Trees, on-chain state limits, etc for that specific example.

Reply:
that's a good point, the intro wasn't particularly friendly 😅